integration_tests/metric_test.py

```diff
@@ -149,3 +150,43 @@ def test_qa_metrics(self) -> None:
         self.assertAlmostEqual(
             overall["F1"].get_value(Score, "score").value, 0.8235975260931867, 2
         )
+
+    def test_sql_exactsetmatch(self):
+        metric = explainaboard.metrics.text_to_sql.SQLExactSetMatchConfig(
+            db_dir="https://expressai-xlab.s3.amazonaws.com/large_data/database",
```

Comment:
@pfliu-nlp : maybe we should have this be on the `inspired-public-data` GCP bucket instead? I don't have a strong opinion either way, but I think we were trying to limit reliance on other data sources to limit possible points of failure.

Reply:
we can move data into gcp inspired-public-data.

Reply:
moved it to GCP bucket